index.html

```diff
@@ -22,5 +22,16 @@
             <div class="line"></div>
         </div>
     </nav>
+    <div class="background">
+        <h1 class="headline">I’m Mariane Glad to see you!</h1>
+        <p class="headline-paragraph">
+            I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
```

Comment:
- [ ] Please use the same margin and padding as in [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design 

- [ ] Please use flexbox to align the elements in the design

| Your design | Figma design | 
| ---------------- | ------------------ |
| ![image](https://user-images.githubusercontent.com/74744787/176566689-ae294651-5a0e-4b28-919c-8289d505c786.png) | ![image](https://user-images.githubusercontent.com/74744787/176566752-7e2b780a-14dc-4103-8871-9cc06fc5777c.png) |